src/PatientTestsApi/Test/Services/PatientDataService.spec.ts

```diff
@@ -25,6 +25,30 @@ describe("PatientDataService #integration", async function (): Promise<void> {
     expect(createdPatient._shardKey).is.equal(id);
   }); 
 
+  it("Won't find a patient that does not exist", async function (): Promise<void> {
+    const dataService: PatientDataService = createPatientDataService();
+    const id = '-123';
+
+    // test
+    const createdPatient = await dataService.findPatient(id);
+    expect(createdPatient).to.be.null;
+  }); 
+
+  it("Can find a patient", async function (): Promise<void> {
+    const dataService: PatientDataService = createPatientDataService();
+    const expectedPatient = PatientFixture.createPatient();
+    expectedPatient.id = 'newId';    
+    const id = await dataService.insertPatient(expectedPatient);
+
+    // test
+    const createdPatient = await dataService.findPatient(id);
+    Object.keys(expectedPatient).forEach(key => {
+      expect(createdPatient[key]).deep.equal(expectedPatient[key]);
+    });
+    expect(createdPatient._id).is.equal(id);
+    expect(createdPatient._shardKey).is.equal(id);
+  }); 
```

Comment:
findPatient should not be returning database properties. (_id and _shardKey)
We can merge the find and create tests into one test called "Can find and create a patient.". Pull the following 
```
const createdPatient = await db.loadPatient(id);
    Object.keys(expectedPatient).forEach(key => {
      expect(createdPatient[key]).deep.equal(expectedPatient[key]);
    });
    expect(createdPatient._id).is.equal(id);
    expect(createdPatient._shardKey).is.equal(id);
```
from the create test, and use this to check the database properties. Also update the assertions to check the database properties are not returned.